cypress/e2e/06-self_registration.cy.js

```diff
@@ -1,10 +1,39 @@
-import { loginHooks, generatePatient } from "../support/e2e";
+import { loginHooks, generatePatient, testNumber } from "../support/e2e";
+import {
+  cleanUpPreviousRunSetupData,
+  setupRunData,
+  cleanUpRunOktaOrgs,
+} from "../utils/setup-utils";
 
 const patient = generatePatient();
+const specRunName = "spec06";
+const currentSpecRunVersionName = `${testNumber()}-cypress-${specRunName}`;
 
 describe("Patient self registration", () => {
   loginHooks();
-  it("gets the self registration link and navigates to it", () => {
+  before("store patient info", () => {
+    cy.task("setPatientName", patient.fullName);
+    cy.task("setPatientDOB", patient.dobForPatientLink);
+    cy.task("setPatientPhone", patient.phone);
```

Comment:
Once the other tests are all isolated, do we want to clean up these getter and setter tasks?